### PR TITLE
Add support for describing multiple inputs for the same content type

### PR DESCRIPTION
**Pull request by cezarystepkowski**

### CODE REVIEW

File: src/Describer/InputDescriber/FormInputDescriber.php [hunk 1]
```diff
@@ -42,12 +44,54 @@ public function describe(Input $input, Operation $operation, string $httpMethod)
...
         } elseif ($input->isInBody()) {
-            $operation->requestBody = new RequestBody(
-                [
-                    'required' => true,
-                    'content' => $this->bodyDescriber->describe($form, $httpMethod),
-                ]
-            );
+            $describedBody = $this->bodyDescriber->describe($form, $httpMethod);
+
+            if ($operation->requestBody === null) {
```
Comment (dragosprotung) [inline src/Describer/InputDescriber/FormInputDescriber.php:49]: We should split this method and do some early returns. It's gotten quite big.

File: src/Describer/InputDescriber/FormInputDescriber.php [hunk 2]
```diff
@@ -42,12 +44,54 @@ public function describe(Input $input, Operation $operation, string $httpMethod)
...
+                $operation->requestBody = new RequestBody(
+                    [
+                        'required' => true,
+                        'content' => $describedBody,
+                    ]
+                );
+            } else {
+                Assert::isInstanceOf($operation->requestBody, RequestBody::class);
```
Comment (dragosprotung) [inline src/Describer/InputDescriber/FormInputDescriber.php:57]: Is this check rally needed ?

**Review by dragosprotung — CHANGES_REQUESTED** [review]

File: src/Describer/InputDescriber/FormInputDescriber.php (hunk 2 above)
Comment (cezarystepkowski) [reply src/Describer/InputDescriber/FormInputDescriber.php:57]: According to docblock `$operation->requestBody` might be also a `Reference` but we don't support this scenario currently.

Comment (dragosprotung) [reply src/Describer/InputDescriber/FormInputDescriber.php:57]: Could you add a comment for this ?

File: src/Describer/InputDescriber/FormInputDescriber.php (hunk 1 above)
Comment (cezarystepkowski) [reply src/Describer/InputDescriber/FormInputDescriber.php]: I've cleaned up the code, should be more readable now.

**Review by dragosprotung — APPROVED** [review]

**Pull request merged**